kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/OffsetAcker.java

```diff
@@ -74,8 +88,31 @@ public void ackOffsets(String groupId, Map<TopicPartition, OffsetAndMetadata> of
                     log.warn("Error when get consumer for offset ack:", throwable);
                     return;
                 }
-                MessageId messageId = MessageIdUtils.getMessageId(offsetAndMetadata.offset());
-                consumer.acknowledgeCumulativeAsync(messageId);
+                KopTopic kopTopic = new KopTopic(topicPartition.topic());
+                String partitionTopicName = kopTopic.getPartitionName(topicPartition.partition());
+                brokerService.getTopic(partitionTopicName, false).whenComplete((topic, error) -> {
```

Comment:
Should we add some logs to handle the case that `error` is not null or `topic` is `Optional.empty()`?